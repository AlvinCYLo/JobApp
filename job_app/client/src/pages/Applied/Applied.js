import React from 'react';

const Applied = () => {
  return <div>APPLIED</div>;
};

export default Applied;
