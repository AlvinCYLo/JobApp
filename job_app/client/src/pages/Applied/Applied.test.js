import React from 'react';
import renderer from 'react-test-renderer';
import Applied from './Applied';

it('renders Applied page correctly', () => {
  const tree = renderer.create(<Applied />).toJSON();
  expect(tree).toMatchSnapshot();
});
