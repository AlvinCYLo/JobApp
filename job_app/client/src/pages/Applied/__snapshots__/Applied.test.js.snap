// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Applied page correctly 1`] = `
<div>
  APPLIED
</div>
`;
